azurerm/internal/services/storage/tests/resource_arm_storage_account_test.go

```diff
@@ -977,6 +1020,93 @@ resource "azurerm_storage_account" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomString)
 }
 
+func testAccAzureRMStorageAccount_minTLSVersion10(data acceptance.TestData) string {
```

Comment:
Can we encapsulate these 3 functions into one, and control the version under used via a variable?

Reply:
Changed to one function accepting `tlsVersion string` and checking it.